molo/core/content_import/views.py

```diff
@@ -30,12 +31,11 @@ def get_repo_languages(request, name):
     ws = RemoteWorkspace('%s/repos/%s.json' % (
         settings.UNICORE_DISTRIBUTE_API, name))
     ws.sync(Localisation)
+    locales, errors = ContentImportHelper(ws).parse_locales()
 
     return Response({
-        'locales': [{
-            'locale': l.locale,
-            'name': Locale.parse(l.locale).english_name
-        }for l in ws.S(Localisation).all()],
+        'locales': locales,
+        'errors': errors
     })
 
 
```

Comment:
Should we also be updating the views import and validate views to catch `ImportError`s and return the appropriate responses?